core/src/main/java/greencity/controller/AchievementController.java

```diff
@@ -47,39 +43,4 @@ public AchievementController(AchievementService achievementService) {
     public ResponseEntity<List<AchievementVO>> getAll() {
         return ResponseEntity.status(HttpStatus.OK).body(achievementService.findAll());
     }
-
-    /**
-     * Method notifies of the achievement.
-     *
-     * @param userId of {@link UserVO}
-     * @return list {@link AchievementNotification}
-     */
-    @ApiOperation(value = "Get all the achievements that need to notify.")
-    @ApiResponses(value = {
-        @ApiResponse(code = 200, message = HttpStatuses.OK),
-        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
-    })
-    @GetMapping("/notification/{userId}")
```

Comment:
are you sure that UI components do not use these endpoints?

Reply:
Yes